core/tests/wdio.conf.js

```diff
@@ -329,6 +329,8 @@ exports.config = {
         '-i', process.env.DISPLAY,
         '-g', '300',
         '-loglevel', '16',
+        // Use ultrafast preset to reduce overhead, curcial when testing in CI.
```

Comment:
Maybe replace "curcial" with "which is crucial", to make the sentence flow better. (Note misspelling of "crucial")

Reply:
Done!